sccm-currentbranch/azuredeploy.json

```diff
@@ -90,19 +48,17 @@
     }
   },
   "variables": {
-    "Quotes": "\"",
-    "storageAccountName": "[concat(uniqueString(resourceGroup().id),'storage')]",
-    "containerName": "[concat(toLower(parameters('prefix')), 'container')]",
-    "vnetId": "[resourceId(parameters('existingVnetResourcegroup'), 'Microsoft.Network/virtualNetworks', parameters('exsitingVnetName'))]",
-    "generalSettings": {
-      "storageAccountType": "Standard_LRS",
-      "diskstorageAccountType": "Premium_LRS",
-      "virtualMachinessize": "Standard_D2s_v3"
-    },
+    "dscScript": "DSC/DCConfiguration.zip",
+    "virtualNetworkName": "[concat(toLower(parameters('prefix')), '-vnet')]",
+    "vnetId": "[resourceId(resourceGroup().name, 'Microsoft.Network/virtualNetworks', variables('virtualNetworkName'))]",
+    "domainName": "contoso.com",
```

Comment:
seems like this should be a parameters (with defaultValue)?

Reply:
It's for demo or lab purpose. We don't want user to customize it. So I put general name contoso.com here.